proxy/envoy/testdata/envoy-sidecar-auth.json.golden

```diff
@@ -0,0 +1,120 @@
+{
+  "listeners": [],
+  "lds": {
+    "cluster": "lds",
+    "refresh_delay_ms": 10
+  },
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": "tcp://127.0.0.1:15000"
+  },
+  "cluster_manager": {
+    "clusters": [
+      {
+        "name": "rds",
+        "connect_timeout_ms": 1000,
+        "type": "strict_dns",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://localhost:15003"
+          }
+        ],
+        "ssl_context": {
+          "cert_chain_file": "/etc/certs/cert-chain.pem",
+          "private_key_file": "/etc/certs/key.pem",
+          "ca_cert_file": "/etc/certs/root-cert.pem",
+          "verify_subject_alt_name": [
+            "spiffe://cluster.local/ns/istio-system-auth/sa/istio-pilot-service-account"
```

Comment:
What is this namespace ? We should test with the ns where the default install is using (istio-system)